databricks/README.md

```diff
@@ -1,9 +1,15 @@
 # Agent Check: Databricks
 
+![Databricks default dashboard][21]
+
 ## Overview
 
 Monitor your [Databricks][1] clusters with the Datadog [Spark integration][2].
 
+This integration unifies logs, infrastructure metrics, and Spark performance metrics, providing real-time visibility into the health of your nodes and the performance of your jobs. It can help you debug errors, fine-tune performance, and identify issues such as inefficient data partitioning or clusters running out of memory.
+
+For feature details, see [Monitor Databricks with Datadog][22].
```

Comment:
Just a small suggestion to add the blog link to the Further Reading section instead, which is where they usually go.

Alternatively, that section could be removed from this page, since there's nothing in it currently:
https://docs.datadoghq.com/integrations/databricks/?tab=driveronly#further-reading